src/main/java/org/broadinstitute/hellbender/tools/PrintReads.java

```diff
@@ -13,7 +13,33 @@
 import org.broadinstitute.hellbender.utils.read.SAMFileGATKReadWriter;
 
 /**
- * Print reads from a SAM/BAM/CRAM file.
+ * Copy reads in a SAM/BAM/CRAM file verbatim to a SAM file
+ *
+ * <p>
+ *     This tool copies reads from a SAM/BAM/CRAM file to a SAM file. If you are interested in printing reads in
+ *     certain intervals only, use the -L argument.
```

Comment:
@sooheelee Is it worth mentioning that this gets used with BQSR or that it can be used with read filters?  Otherwise it sounds like a pretty pointless tool.

Reply:
@ldgauthier I think ApplyBQSR is intended to be the replacement for using PrintReads for BQSR. So I'm not sure this has much real analytical value, other than filtering (and maybe format conversion, and index generation, but since it has a default filter WellFormed its not necessarily full fidelity by default).

Reply:
I forgot about ApplyBQSR.  Fair point.